lib/tracer.js

```diff
@@ -30,8 +32,12 @@ function tracer() {
 function start() {
   assert(!TRACER);
   // This is delayed to allow process.env to be modified up until start time.
-  var config = configure(null, null, {}, process.env);
-  options.accountKey = config.appName;
+  // var config = configure(null, null, {}, process.env);
+  // FIX ME with appropriate app name.
+  var cwd = process.cwd();
+  var pkgjson = require(cwd + '/package.json');
+  options.accountKey = process.env.STRONGLOOP_APPNAME ||
+    process.env.SL_APP_NAME || pkgjson.name;
```

Comment:
this got copied from strong-agent to get the job done, but we can probably pass this value in a more delicate way.